library/src/main/scala/scala/scalajs/runtime/WasPublicBeforeTyper.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalajs.runtime
+
+import scala.annotation.StaticAnnotation
+
+class WasPublicBeforeTyper extends StaticAnnotation
```

Comment:
And: Is `runtime` really the right location for this? This is as little runtime as it gets...